doc/release-notes-empty-template.md

```diff
@@ -32,6 +32,13 @@ Upgrading directly from a version of Bitcoin Core that has reached its EOL is
 possible, but it might take some time if the data directory needs to be migrated. Old
 wallet versions of Bitcoin Core are generally supported.
 
+Running Bitcoin Core binaries on macOS requires self signing.
```

Comment:
This might be a bit confusing, because you don't need to self-sign the main drag & drop release.